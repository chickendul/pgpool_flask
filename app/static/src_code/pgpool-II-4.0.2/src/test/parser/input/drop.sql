version https://git-lfs.github.com/spec/v1
oid sha256:4e979ededb60dd2bbc25e7c1b15943555b312efaa4e542e843b7b8335aeed6e8
size 358
