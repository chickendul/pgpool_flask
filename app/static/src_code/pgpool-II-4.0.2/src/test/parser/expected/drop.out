version https://git-lfs.github.com/spec/v1
oid sha256:b4ff46b64bac86f10e41600b3c1acf6758f82ed354bbbd7a0285e97c941b5c3c
size 399
